infoschema/perfschema/tables.go

```diff
@@ -35,6 +51,32 @@ const (
 	tableNameTiDBProfileGoroutines           = "tidb_profile_goroutines"
 )
 
+var tableIDMap = map[string]int64{
+	tableNameGlobalStatus:                    autoid.SystemSchemaIDFlag | (autoid.PerformanceSchemaTableIDStart + 1),
```

Comment:
It's better to define the `autoid.PerformanceSchemaTableIDStart = autoid.SystemSchemaIDFlag | xxx ` to simplify code.